cantools/database/can/c_source.py

```diff
@@ -1339,8 +1350,13 @@ def _generate_structs(database_name, messages, bit_fields):
 
     return '\n'.join(structs)
 
+def _is_sender(message, node_name):
+    return node_name == None or node_name in message.senders
+
+def _is_receiver(message, node_name):
+    return node_name == None or not node_name in message.senders
```

Comment:
It's best practice to use `is` when comparing to `None`, as in `node_name is None`.